upstairs/src/upstairs.rs

```diff
@@ -901,71 +880,53 @@ impl Upstairs {
     /// the [DsState::Connecting] state with the negotiation state of
     /// [NegotiationState::LiveRepairReady], indicating it needs to be repaired.
     /// If a Downstairs needs to be repaired, try to start repairing it. When
-    /// starting the repair fails, this function will schedule a task to retry
-    /// the repair by setting [Self::repair_check_deadline].
+    /// starting the repair fails, we will retry on the next event.
     ///
     /// If this Upstairs is [UpstairsConfig::read_only], this function will move
     /// any Downstairs from
     /// `DsState::Connecting { state:  NegotiationState::LiveRepairReady, .. }`
     /// back to [DsState::Active] without actually performing any repair.
     pub(crate) fn on_repair_check(&mut self) {
-        info!(self.log, "Checking if live repair is needed");
         if !matches!(self.state, UpstairsState::Active) {
-            info!(self.log, "inactive, no live repair needed");
-            self.repair_check_deadline = None;
             return;
         }
 
         if self.cfg.read_only {
-            info!(self.log, "read-only, no live repair needed");
```

Comment:
At first I was, wait, no I want my messages.
But, then I remembered this now fires every loop, so, yeah, no need to spam the logs..

Reply:
Yup, exactly.